docs/installation-mac.md

```diff
@@ -65,6 +65,14 @@ brew uninstall observiq/observiq-otel-collector/observiq-otel-collector
 
 # If you moved the opentelemetry-java-contrib-jmx-metrics.jar
 sudo rm -f /opt/opentelemetry-java-contrib-jmx-metrics.jar
+
+# To remove configs run:
+# - on Mac M1:
+rm -f /opt/homebrew/etc/observiq_config.yaml
+rm -f /opt/homebrew/etc/observiq_config.yaml.default
+# - on Mac Intel:
+rm -f /usr/local/etc/observiq_config.yaml
+rm -f /usr/local/etc/observiq_config.yaml.default
```

Comment:
Is there a brew command that will give this to you?

Reply:
looks like it's `$(brew --prefix)/etc/observiq_config.yaml`

Reply:
@cpheps yep, a much smarter way to do it. i'll make the changes